rockhound/bedmap2.py

```diff
@@ -24,7 +24,7 @@
 }
 
 
-def fetch_bedmap2(datasets, *, load=True):
+def fetch_bedmap2(datasets, *, load=True, chunks=100, **kwargs):
```

Comment:
I set the default chunks number arbitrarily. Maybe we should increase it to speed up computations by reducing overheads.
Based on [Dask Best Practices on arrays](https://docs.dask.org/en/latest/array-best-practices.html#select-a-good-chunk-size), we could assign chunks size of `(1000, 1000)` taking into account that several datasets may be loaded simultanously.

What do you think?

Reply:
@santisoler sounds good to me. We can test what works best for this dataset.

Reply:
:+1: